packages/compass-query-bar/src/components/query-option.tsx

```diff
@@ -26,8 +26,6 @@ const documentEditorOptionContainerStyles = css({
 });
 
 const queryOptionLabelStyles = css({
-  // A bit of vertical padding so users can click the label easier.
```

Comment:
FWIW if clicking area needs to be extended without disturbing the actual content box you'd usually use an absolute pseudo element not actual padding, that way changing layout doesn't affect other content and you don't need to remove extended clicking area because of the layout changes